prov/efa/src/rdm/rxr_rma.c

```diff
@@ -155,6 +155,10 @@ ssize_t rxr_rma_readmsg(struct fid_ep *ep, const struct fi_msg_rma *msg, uint64_
 	       flags);
 
 	rxr_ep = container_of(ep, struct rxr_ep, base_ep.util_ep.ep_fid.fid);
+	if ( OFI_UNLIKELY(0 == (rxr_ep_domain(rxr_ep)->info->caps & FI_RMA)) ) {
+		return -FI_EOPNOTSUPP;
```

Comment:
Should we add `FI_WARN_ONCE` here?

Reply:
```
rxr_ep_domain(rxr_ep)->info
```
is not the right info object to use. You need to use `rxr_ep->user_info`.

Reply:
Probably add a unit test to verify the behavior

Reply:
unit test added.